__tests__/app.test.js

```diff
@@ -0,0 +1,40 @@
+
+const db = require('../db/connection.js')
+const testData = require('../db/data/test-data')
+const request = require('supertest')
+const app = require('../app.js')
+const seed = require('../db/seeds/seed')
+
+
+beforeEach(() => 
+    seed(testData)
+);
+
+afterAll(()=> 
+     db.end()
+)
+
+
+describe('GET /api', () => {
+    describe('/api/topics', () => {
+        test('200: Returns an array of topic objects with the following properties: slug, description.', () => {
+            return request(app)
+            .get('/api/topics')
+            .expect(200)
+            .then((res) => {
+                expect(res.body.topics.length).toBe(3)
```

Comment:
You have said in your test description that you're expecting a slug and description key but you are not testing for that. This could be any array with a length of 3